build.sbt

```diff
@@ -78,11 +78,9 @@ lazy val root = (project in file("."))
 
 javaOptions in Universal ++= Seq(
   "-Dpidfile.path=/dev/null",
-  "-J-XX:MaxRAMFraction=2",
-  "-J-XX:InitialRAMFraction=2",
+  "-J-XX:MaxRAMPercentage=60",
+  "-J-XX:InitialRAMPercentage=60",
```

Comment:
Rather than 50%, I've moved to 60% - this was desired but never possible under the previous setup. We should consider making this even higher now that we can!

Reply:
Do you mean now that we can because of Java 11 or because we have more RAM on our machines?

Reply:
that's getting risky, aas the 60% is only the heap, and there is something else that it also limits to exactly the same amount, meaning that the JVM can use all the system ram plus more.
I found that information when investigating a system out of memory on support-frontend after it had been running a while.

Reply:
see https://github.com/guardian/support-frontend/pull/2907